src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -279,6 +301,10 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
     return this.options.field.slice(1);
   }
 
+  public get facetType() {
```

Comment:
Mostly for UA, but same enum also used with the API #consistancy

Reply:
But thats on All facets no? There is only one DynamicFacet for now and the DynamicRangeFacet will extend it.

Reply:
Ah, I thought that interface was specific to the new dynamic facets